File: android studio/yongin/app/src/main/java/com/example/yongin_java/MainActivity.java
package com.example.yongin_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.provider.Telephony;
import android.view.View;
import android.widget.AdapterView;
import android.widget.GridView;
import android.content.Intent;


public class MainActivity extends AppCompatActivity {

    GridView gridView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        gridView = (GridView) findViewById((R.id.grid_view));
        gridView.setAdapter(new ImageAdapter(this));

        gridView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            @Override
            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {

                Intent intent = new Intent(getApplicationContext(),FullScreenActivity.class);
                intent.putExtra("id",position);
                startActivity(intent);




            }
        });
    }
}

File: anstu/test/app/src/main/java/com/example/test/MainActivity.kt
package com.example.yongin_java

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.AdapterView.OnItemClickListener
import android.widget.GridView
import androidx.appcompat.app.AppCompatActivity
import com.example.test.FullScreenActivity
import com.example.test.ImageAdapter
import com.example.test.R

class MainActivity : AppCompatActivity() {
    var gridView: GridView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        gridView = findViewById<View>(R.id.grid_view) as GridView
        gridView!!.adapter = ImageAdapter(this)
        gridView!!.onItemClickListener =
            OnItemClickListener { parent, view, position, id ->
                val intent = Intent(applicationContext, FullScreenActivity::class.java)
                intent.putExtra("id", position)
                startActivity(intent)
            }
    }
}